generators/app/templates/app/middlewares/docsAuth.ejs

```diff
@@ -0,0 +1,16 @@
+const logger = require('../logger');
+const { receivedTokenHeader } = require('../../config/constants')
+const { storedToken } = require('../../config/environment')
+
+exports.verifyDocumentationToken = (req, res, next) => {
+  const receivedToken = req.headers[receivedTokenHeader];
```

Comment:
why receivedTokenHeader? with that name nobody knows the proposal of that token
its documentation token or something like that
try to be more explicit